<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioFilters</class>
 <widget class="QWidget" name="AudioFilters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>1600</height>
   </size>
  </property>
  <widget class="QRadioButton" name="RD45_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>RD45</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Wave_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Wave</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Default_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Value_label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlot" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>248</width>
     <height>141</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>1600</height>
    </size>
   </property>
  </widget>
  <widget class="QSlider" name="Value_indicator">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>25</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="setButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../sharedlib/header/qcustomplot.h</header>
   <container>1</container>
   <slots>
    <slot>graph()</slot>
    <slot>slot2()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>RD45_button</sender>
   <signal>clicked()</signal>
   <receiver>AudioFilters</receiver>
   <slot>RD45()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Wave_button</sender>
   <signal>clicked()</signal>
   <receiver>AudioFilters</receiver>
   <slot>Wave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Default_button</sender>
   <signal>clicked()</signal>
   <receiver>AudioFilters</receiver>
   <slot>Default()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>RD45()</slot>
  <slot>Wave()</slot>
  <slot>Default()</slot>
  <slot>Graph()</slot>
  <slot>slot1()</slot>
 </slots>
</ui>
